src/React.Core/INodeJsEngine.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Reflection;
+
+namespace React
+{
+	/// <summary>
+	/// The wrapper for the Node engine
+	/// </summary>
+	public interface INodeJsEngine
+	{
+		string Name { get; }
+		string Version { get; }
+
+		bool HasVariable(string uSER_SCRIPTS_LOADED_KEY);
```

Comment:
Is it intentional that `u` is lowercase?  🤔

Reply:
Hah, the Visual Studio code-generator couldn't decide how to handle the screaming snake case